openapi.yml

```diff
@@ -12821,11 +12821,9 @@ components:
         relevant:
           type: boolean
         type:
-          type: string
-          readOnly: true
+          $ref: '#/components/schemas/LabelTypeEnum'
       required:
       - label
-      - type
```

Comment:
`type` is not required on POST but it is always provided, so it's required on GET.

maybe instead of making the serializer field `required=False` it could be `required=True` but have a default to `CONTEXT_BASED` or `ALIAS`? It only matters for Alias anyway.